src/runtime_src/core/tools/xbtop/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2022 Xilinx, Inc. All rights reserved.
+#
+
+message(${LINUX_FLAVOR} ${LINUX_VERSION})
+string(SUBSTRING "${LINUX_VERSION}" 0 1 major_version)
+
+if (${LINUX_FLAVOR} STREQUAL "Amazon")
+  set (PYTHON_3 FALSE)
+elseif ((${LINUX_FLAVOR} STREQUAL "CentOS") AND (${major_version} STREQUAL "7"))
+  set (PYTHON_3 FALSE)
+endif ()
+
+if (${PYTHON_3})
```

Comment:
Hi @AShivangi  ,
The logic seems like install() for xbtop are excluded if distro is CentOS 7.x
Will this affect behavior of `xbtop` on CentOS 7.x ? Or this change is due to python version being used on Centos ?

Reply:
I'll discuss this with you offline